mgnify_pipelines_toolkit/analysis/shared/dwc_summary_generator.py

```diff
@@ -0,0 +1,243 @@
+import argparse
+from collections import defaultdict
+import pathlib
+import logging
+import requests
+
+import pandas as pd
+import pyfastx
+
+logging.basicConfig(level=logging.DEBUG)
+
+url = "https://www.ebi.ac.uk/ena/portal/api/search?result"
+runs_url = f"{url}=read_run&fields=secondary_study_accession%2Csample_accession&limit=10&format=json&download=false"
```

Comment:
```suggestion
runs_url = f"{url}=read_run&fields=secondary_study_accession,sample_accession&limit=10&format=json&download=false"
```
No need to URL encode the string here, requests will do it for you

Reply:
I think...

Reply:
tested it and it works, thanks :D